frontend/src/components/module-editor/EditContentOptionsMenu.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { HStack, IconButton } from "@chakra-ui/react";
+import { EditIcon, CopyIcon, DeleteIcon } from "@chakra-ui/icons";
+import { EditContentOptionsMenuProps } from "../../types/ModuleEditorTypes";
+
+const EditContentOptionsMenu = ({
+  isVisible,
+  onEditClick,
+  onCopyClick,
+  onDeleteClick,
+}: EditContentOptionsMenuProps): React.ReactElement => {
+  const currentTabIndex = isVisible ? 0 : -1;
+
+  return (
+    <HStack spacing={4} opacity={isVisible ? 1 : 0}>
+      <IconButton
+        aria-label="Edit this content block"
+        tabIndex={currentTabIndex}
+        variant="transparent"
+        onClick={onEditClick}
+        m={0}
+        icon={<EditIcon color="background.dark" h={5} w={5} />}
+      />
```

Comment:
(optional) We could extract this into a separate component potentially:
```suggestion
      <TransparentButton
        label="Edit this content block"
        tabIndex={currentTabIndex}
        onClick={onEditClick}
        icon={EditIcon}
      />
```

Reply:
Hmm I may leave it for now - I could see it cleaning up within this file, I'm not sure creating a transparent button with these stylings has much application (that isn't already handled by the new `button.ts` styling) otherwise